cob_hardware_config/cob4-10/config/joint_state_publisher.yaml

```diff
@@ -1,2 +1,2 @@
 rate: 100 #Hz
-source_list: [/base/joint_states, /torso/joint_states, /sensorring/joint_states, /head/joint_states]
+source_list: [/base/joint_states, /torso/joint_states, /head/joint_states]
```

Comment:
`/head/joint_states` contains head+sensorring.
The relay is needed just for the script server, which expects them in sensorring as well.